src/context/Video/Watchlater.jsx

```diff
@@ -32,16 +33,18 @@ export const addToWatchlater = async (video, videoDispatch) => {
     );
     const { status, data } = response;
     if (status === 200 || status === 201) {
-      videoDispatch({ type: "ADD_TO_WATCHLATER", payload: data?.watchlater });
-      Toast({ message: "Added to Watchlist", type: "success" });
+      userDispatch({ type: "ADD_TO_WATCHLATER", payload: data?.watchlater });
+      toast.success("Saved to Watchlist");
+    } else if (status === 409) {
+      toast.error("Video already saved to watch later");
     }
   } catch (error) {
-    Toast({ message: "Couldn't add to Watchlist", type: "error" });
+    toast.error("Something went wrong");
     console.log(error);
   }
 };
 
-export const removeFromWatchlater = async (video, videoDispatch) => {
+export const removeFromWatchlater = async (video, userDispatch) => {
   try {
     const response = await axios.delete(`/api/user/watchlater/${video._id}`, {
```

Comment:
same above

Reply:
Thank You. Will do that during refactoring.